### PR TITLE
Move HA e2e logs to separate folders

### PR DESCRIPTION
**Pull request by as51340**
### Description

It's hard to inspect one log file for multiple tests when something goes wrong. Hence, move e2e logs to separate folders. 

[master < Task] PR
- [x] Provide the full content or a guide for the final git message
    - **Move HA e2e logs to separate folders**

### Documentation checklist
- [x] Add the documentation label tag
- [x] Add the bug / feature label tag
- [x] Add the milestone for which this feature is intended
    - If not known, set for a later milestone
- [ ] Write a release note, including added/changed clauses
    - **[Release note text]**
- [ ] Link the documentation PR here
    - **[Documentation PR link]**
- [x] Tag someone from docs team in the comments

### CODE REVIEW

**Comment by MarkoBarisic** [comment]
How does this PR effect [this step](https://github.com/memgraph/memgraph/blob/93962b1a79e34d8ebac8b7420da984610a5144b7/.github/workflows/diff_release.yaml#L302) in `diff` and [this function](https://github.com/memgraph/memgraph/blob/93962b1a79e34d8ebac8b7420da984610a5144b7/release/package/mgbuild.sh#L443) in `mgbuild.sh`. 
In short, will the name of the logs file to upload to github change?

**Review by MarkoBarisic — APPROVED** [review]
LGTM

File: tests/e2e/high_availability/coord_cluster_registration.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def get_instances_description():
                 "--management-port",
                 "10011",
             ],
-            "log_file": "instance_1.log",
+            "log_file": "high_availability/coord_cluster_registration/instance_1.log",
```
Comment (antoniofilipovic) [inline tests/e2e/high_availability/coord_cluster_registration.py:43]: Can you put this as some kind of constant in the file

Comment (antoniofilipovic) [reply tests/e2e/high_availability/coord_cluster_registration.py:43]: coord_cluster_registration as constant

**Review by antoniofilipovic — COMMENTED** [review]
Looks good, I would put folder names as constants in files

File: tests/e2e/high_availability/coord_cluster_registration.py (hunk 1 above)
Comment (as51340) [reply tests/e2e/high_availability/coord_cluster_registration.py:43]: I think it is okay like this

**Pull request merged**